sdk/src/types/api/core/response.rs

```diff
@@ -128,20 +129,62 @@ pub struct SubmitBlockResponse {
     pub block_id: BlockId,
 }
 
-/// Describes the ledger inclusion state of a transaction.
+/// Describes the state of a block.
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
 #[cfg_attr(
     feature = "serde",
     derive(serde::Serialize, serde::Deserialize),
     serde(rename_all = "camelCase")
```

Comment:
rename not needed?

Reply:
I think it doesn't hurt to have them tbh, in case we add fields where it actually applies in the future